x/assets/keeper/expected_keepers.go

```diff
@@ -0,0 +1,10 @@
+package keeper
```

Comment:
Please move this to `x/assets/types/expected_keepers.go` and refer to it from there.

Reply:
That's two different approaches, if we put `expected_keepers.go`, those interfaces define should use module-type-unrelated structure as input/response. But we currently not do that well, so if put into types, that would led to cycle import. Put this in keeper and these interfaces define will only import types(and types will only be imported not import other package), and then get no cycle import

Reply:
I see; please add a comment here saying that doing this avoids import cycle.

Reply:
added